_data/sites.json

```
@@ -11617,6 +11617,19 @@
         ]
     },
 
+    {
+        "name": "Langfuse",
+        "url": "https://langfuse.com/support",
+        "difficulty": "hard",
+        "email": "security@langfuse.com",
+        "notes" : "Contact [security@langfuse.com](mailto:security@langfuse.com) and ask for your account to be deleted.",
```

```suggestion
        "notes" : "Contact them via e-mail and ask for your account to be deleted.",
```
We don't repeat the e-mail on the notes.